server/api/muster/index.ts

```diff
@@ -18,17 +18,17 @@ router.get(
 );
 
 router.get(
-  '/:orgId/individuals',
+  '/:orgId/roster',
```

Comment:
I've struggled with this since the beginning. `Roster` refers to both a single `RosterEntry` as well as a collection of them. I was never necessarily a fan of `individual` but it would be nice for us to get consistency around this... good first step here. You rock.

Reply:
That Roster naming has always driven me crazy too. I've tried to keep the variable naming consistent at least by using `rosterEntry`. That's probably what the actual table/model should be changed to.